docs/preview/features/logging.md

```diff
@@ -138,6 +139,33 @@ So the resulting log message becomes:
 
 `HTTP Request POST http://localhost:5000/weatherforecast completed with 200 in 00:00:00.0191554 at 03/23/2020 10:12:55 +00:00 - (Context: [X-Api-Key,])`
 
+## Tracking application version
+
+The `Arcus.WebApi.Logging` library allows you to add application version tracking to your <span>ASP.NET</span> application which will include your application version to a configurable response header.
+
+This functionality uses the `IAppVersion`, available in the [Arcus.Observability]() library, for retrieving the current application version.
```

Comment:
Is this link intentionally left out?

Reply:
Forgotton, I'll place in the general root docs link here as we're not yet done with the app version in the observability. I guess.